runtime/src/main/java/org/corfudb/runtime/object/transactions/AbstractTransactionalContext.java

```diff
@@ -79,6 +79,7 @@
     public final TransactionBuilder builder;
 
     /**
+     * TODO remove this!
```

Comment:
Why remove it? It seems like it may be useful to keep track of the running time of a transaction - and either way shouldn't have much overhead.